### DISCUSSION TITLE
I am suddenly getting ?(caption) in my CSV generated file tables

### DISCUSSION THREAD

**Discussion by IceManTBM3** [quarto-dev/quarto-cli#6200] [Q&A]
### Description

is this Quarto or R?

Running on Win 11

check dump:
tmeek@AC-LT-TMEE-01 MINGW64 /c/Doc-Testing/diit-documentation/quarto (Tom_Doc_2)
$ quarto check
A new release of Deno is available: 1.28.2 → 1.35.0 Run `deno upgrade` to install it.

[>] Checking versions of quarto binary dependencies...
      Pandoc version 3.1.1: OK
      Dart Sass version 1.55.0: OK
[>] Checking versions of quarto dependencies......OK
[>] Checking Quarto installation......OK
      Version: 1.3.353
      Path: C:\Users\tmeek\AppData\Local\Programs\Quarto\bin
      CodePage: 1252

[>] Checking basic markdown render....OK

[>] Checking Python 3 installation....OK
      Version: 3.10.5
      Path: C:/Users/tmeek/AppData/Local/Programs/Python/Python310/python.exe
      Jupyter: (None)

      Jupyter is not available in this Python installation.
      Install with py -m pip install jupyter

[>] Checking R installation...........OK
      Version: 4.2.2
      Path: C:/Users/tmeek/AppData/Local/Programs/R/R-4.2.2
      LibPaths:
        - C:/Doc-Testing/diit-documentation/quarto/renv/library/R-4.2/x86_64-w64-mingw32
        - C:/Users/tmeek/AppData/Local/Temp/RtmpyUFVfa/renv-system-library
      knitr: 1.39
      rmarkdown: 2.14

Code in my set up file

## NRI Risk Features

<BR>

::: column-margin
::: callout-tip
Column headers are sortable on click. Also, you can filter columns by typing text into the text boxes under each column.
:::
:::

::: panel-tabset
## Field Definitions
```{r}
#| label: tbl-nri-risk-definitions
#| tbl-cap: "NRI Risk Table Fields"
#| echo: false
library(reactable)
library(reactablefmtr)
reactable(dict%>%filter(Feature_Family == "NRI",Release_Status == "prod")%>%               # 1) Choose Feature Family Here
          select(                                                                                # 2) Choose Fields Here and (8)
            Display_Name,
            Release_Status,
            Description,
            Field_Name,
            Units, 
            Feature_Family,
            Feature_Sub_Family,
            Attribution), 
          defaultSorted = list(Field_Name = "asc"),                       # 3) Choose Starting Sort Here
          style = list(fontFamily = 'Arial',fontSize = '12px'),                                  # 4) Choose Table Formatting
          filterable = TRUE,                                                                     # 5) Choose Display Options
          defaultPageSize = 1000,
          highlight = TRUE,
          showPageSizeOptions = TRUE,
          pageSizeOptions = c(20, 50, 100,1000),
          fullWidth = TRUE,
          columns = list(
              Display_Name       = colDef(name = "Display Name"),
              Release_Status     = colDef(name = "Release Status",     show=FALSE, cell = prodCheck, width = 70),
              Description        = colDef(name = "Description", minWidth = 120), 
              Field_Name         = colDef(name = "Field Name (Feature)"),
              Feature_Family     = colDef(name = "Feature Family",     show=FALSE),
              Feature_Sub_Family = colDef(name = "Feature Sub Family", minWidth = 70),
              Units              = colDef(name = "Units",              show = FALSE, minWidth = 70),
              Attribution        = colDef(name = "Attribution",        html = TRUE, cell = nriLinker)
          )
)
```

I haven't really modified anything to significantly but now am getting this random ?(caption) and can't figure out where to go to nip it. Thanks for any help in advance

![image](https://github.com/quarto-dev/quarto-cli/assets/129208946/20db15a7-1b3f-425d-882a-e9a36a06b00d)

**Comment by IceManTBM3** [comment]
PS these tables are created from Excel sheets

**Comment by mcanouil** [comment]
Using `label: tbl-*` means create cross-referenceable table (thus remove it for no cross-reference), see <https://quarto.org/docs/authoring/cross-references.html#tables>.

- #6201

**Reply by IceManTBM3** [reply]
thx will try
